_includes/datetime.html

```diff
@@ -9,6 +9,17 @@
   >
     {{ include.event.date | date:include.date_only_format }}
   </time>
+  {% if include.event.end_date %}
+    &mdash;
+    <time
+      {{ cancelled }}
+      class="time-span"
```

Comment:
I don't think this should be a `time-span`. That class is used by the localisation JS to localise the times of the event. If we're in the `time_tbc` branch there isn't a time set yet and so there's nothing to localise.
```suggestion
```